src/parser/DefaultParser.js

```diff
@@ -29,6 +29,16 @@ class DefaultParser {
     this.pluginData.parseErrors = [];
   }
 
+  /**
+   * Get the list of models from all parsable files.
+   *
+   * @param {FileInformation[]} [files=[]] - List of parsable files.
+   * @return {FileInformation[]} List of Folder or file path that represents a model.
```

Comment:
List of folder and file paths that represent a model.

Reply:
done